pkg/infrastructure/services/resources.go

```diff
@@ -197,10 +200,13 @@ func (s *serviceDeployer) applyKafkaConfigurations(deployment *appsv1.Deployment
 	if len(URI) > 0 {
 		framework.SetEnvVar(enableEventsEnvKey, "true", &deployment.Spec.Template.Spec.Containers[0])
 		for _, kafkaTopic := range s.definition.KafkaTopics {
-			framework.SetEnvVar(fromKafkaTopicToQuarkusEnvVar(kafkaTopic), URI, &deployment.Spec.Template.Spec.Containers[0])
+			appProps[fromKafkaTopicToQuarkusAppProp(kafkaTopic)] = URI
```

Comment:
We set Quarkus even for SpringBoot apps ?

Reply:
We should not.

Reply:
Corrected. Thanks.